app/helpers/events_helper.rb

```diff
@@ -70,6 +70,15 @@ def visit_request_please_text(visit_request)
     t('visit_requests.messages.registration_closed').html_safe
   end
 
+  def get_counter(event)
+    "#{event.limit_newbies} #{t('events.free_places')}"
+  end
+
+  def get_limit_percent(event)
+    return 0 if event.limit_total.zero?
```

Comment:
Can we add an empty line between the guard clause and calculation logic?
Other than that - LGTM 👍

Reply:
Does this really matters? ...

Reply:
This is a not a big deal but it may help with the readability
https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods